src/main/app.ts

```diff
@@ -48,18 +50,20 @@ glob.sync(__dirname + '/routes/**/*.+(ts|js)')
 
 setupDev(app,developmentMode);
 // returning "not found" page for requests with paths not resolved by the router
-app.use((req, res) => {
+app.use((req: any, res) => {
```

Comment:
might be best to also give res a type to keep it consistent

Reply:
Done